thirdparty/maven/zimbra-maven/debian/watch

```diff
@@ -0,0 +1,2 @@
+version=3
+http://archive.apache.org/dist/maven/maven-3/3.3.9/source/apache-maven-(\d+\.\d+\.\d+).tar.xz
```

Comment:
Hmm, ideally we don't have that hard coded 3.3.9 here.   Wonder if we want to pre-process the watch file to substitute something like PVER here.